torch/ao/quantization/fx/match_utils.py

```diff
@@ -49,16 +75,30 @@ def is_match(modules, node, pattern, max_uses=sys.maxsize):
     if len(node.users) > max_uses:
         return False
 
+    if (
+        # This is True if we are not at the last node in the pattern
+        max_uses < sys.maxsize and
```

Comment:
could you explain a little bit more about what does this mean and why do we need this check?

Reply:
you are right, this check is incorrect, if the last node in the pattern is using a module which is used by other nodes we also cannot fuse without further analysis.  Will remove this.